packages/victory-core/src/victory-accessibility-group/victory-accessibility-group.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class VictoryAccessibilityGroup extends React.Component {
+  static displayName = "VictoryAccessibilityGroup";
+  static role = "container";
```

Comment:
Since you've added `role = "container"`, this component will have access to a bunch of additional props added by its parent when it's used on a component like `VictoryBar` rather than on one like `VictoryGroup` is in your demo. This should let you come up with a descriptive id more easily when you need one.

Reply:
To have it work identically with `VictoryGroup`, `VictoryStack`,  and `VictoryChart` though, some additional work would be needed on those components so ensure they're passing the same props to their groupComponent that they pass to their `containerComponent` if it has `role = "container"`

Reply:
gotcha, is that the preference or should the role prop be removed?

Reply:
It depends on what you end up doing as a backup `descId`. I think they only thing you might want from those props would be the key / id that Victory generates for children of a chart that don't have explicit names. You'd end up with something like `chart-group-0-bar-1-parent` for example, which would be useful. If you end up going with a different solution for `descId`, and don't need those props, you should ditch the `container` role.

Reply:
I see, so I could allow `descId` unrequired `if(desc)` and generate it from children. Cool I will look at this now

Reply:
Yeah, you might try adding `VictoryAccessibleGroup` to something like `VictoryBar` to see what you're working with. I can help you figure out how to get `VictoryGroup`, `VictoryStack` and `VictoryChart` to also pass these props to their `container` `groupComponents`

Reply:
Would something this work or should we add/pass a `name` (or `id`/preferred object prop) to components like `VictoryGroup` and `VictoryStack`?

```
    const descId =
      this.props["aria-describedby"] ||
      this.props.name ||  
      this.props.children.map((prop) => prop.props.name).join(", ")  || 
      desc;
```

Reply:
Oh! Actually, I really like the solution of using the `desc` string itself as a the id if one isn't given. Maybe kebab cased? Then we can remove the `container` role and avoid passing extra props.

Reply:
oh great! that makes it easy. Have made this change, just adding tests then hopefully ready for final review